src/pages/venues/[id]/map/index.tsx

```diff
@@ -0,0 +1,144 @@
+import { ApolloProvider, isApolloError, useQuery } from "@apollo/client";
+import { IconArrowLeft } from "hds-react";
+import { GetStaticPropsContext } from "next";
+import dynamic from "next/dynamic";
+import React from "react";
+
+import { VENUE_QUERY } from "..";
+import Page from "../../../../common/components/page/Page";
+import initializeCmsApollo from "../../../../domain/clients/cmsApolloClient";
+import initializeNextApiApolloClient, {
+  useNextApiApolloClient,
+} from "../../../../domain/clients/nextApiApolloClient";
+import Link from "../../../../domain/i18n/router/Link";
+import useRouter from "../../../../domain/i18n/router/useRouter";
+import { getLocaleOrError } from "../../../../domain/i18n/router/utils";
+import serverSideTranslationsWithCommon from "../../../../domain/i18n/serverSideTranslationsWithCommon";
+import { staticGenerationLogger } from "../../../../domain/logger";
+import styles from "./map.module.scss";
+
+const MapView = dynamic(
+  () => import("../../../../common/components/mapView/MapView"),
+  {
+    ssr: false,
+  }
+);
+
+const VenueMapPageContent: React.FC = () => {
```

Comment:
You are probably aware, but using `React.FC` is a topic of a bit of division. Our team stopped using it a year ago when CRA did: https://github.com/facebook/create-react-app/pull/8177